src/main/scala/mesosphere/marathon/core/task/Task.scala

```diff
@@ -61,11 +61,14 @@ sealed trait Task {
 
   def runSpecId: PathId = taskId.runSpecId
 
+  def status: Task.Status
```

Comment:
since the status is now a member of Task, would it be easily possible to remove it from Launched?